know-your-computer/readme.md

```diff
@@ -0,0 +1,34 @@
+# Know Your Computer
+
+"Know Your Computer" focuses on how to perform regular tasks on your computer. In this workshop you will use basic Linux commands in the terminal. The goal is to reduce your fear of using the terminal. 
+
+## Pre-requisite
+To be able to do this workshop you will need a computer with a Linux shell running on it. If you already have a Mac or an Ubuntu operating system or any other Linux distribution running on your laptop, then you are fine. If your laptop is running on Windows operating system, download and install [Git Bash for Windows](https://git-scm.com/downloads).
+Code Your Future does not support Windows, so you must install or dual boot Linux to take our course. But to do this workshop, you can just use Git Bash.
+Once installed, open git bash and you should be able to perform the tasks below. 
+
+## Tasks
+You must use the terminal and Linux commands to complete each challenge. You must not use your GUI! Hands off the mouse! 🖱️ 
+
+ - Create a folder called my-shopping inside your Documents folder.
+ - Then create a file called my-list.txt inside my-shopping folder.
+ - Write the following items to my-list.txt file:
+ ```
+   1. Tea
+   2. Pasta
+   3. Coffee
+   4. Milk
+ ```
+- Create a new folder called my-new-shopping inside your Documents folder.
+- Now move my-list.txt file that is already inside my-shopping folder into my-new-shopping folder so that it does not exist anymore in my-shopping folder.
+- Make a copy of the my-list.txt file that is inside my-new-shopping folder back to my-shopping folder.
```

Comment:
Copy `my-list.txt` from your `my-new-shopping` folder back to your `my-shopping` folder.  After this step, **both** folders should contain a file called `my-list.txt`.